crates/oxc_transformer/src/regexp/options.rs

```diff
@@ -0,0 +1,88 @@
+use crate::env::{can_enable_plugin, Versions};
+
+#[derive(Default, Debug, Clone, Copy)]
+pub struct RegExpOptions {
+    /// Enables plugin for transform the RegExp literal has `y` flag
+    pub sticky_flag: bool,
+    /// Enables plugin for transform the RegExp literal has `u` flag
+    pub unicode_flag: bool,
+    /// Enables plugin for transform the RegExp literal has `s` flag
+    pub dot_all_flag: bool,
+    /// Enables plugin for transform the RegExp literal has `(?<=)` or `(?<!)` lookbehind assertions
+    pub look_behind_assertions: bool,
+    /// Enables plugin for transform the RegExp literal has `(?<name>x)` named capture groups
+    pub named_capture_groups: bool,
+    /// Enables plugin for transform the RegExp literal has `\p{}` and `\P{}` unicode property escapes
+    pub unicode_property_escapes: bool,
+    /// Enables plugin for transform `d` flag
+    pub match_indices: bool,
+    /// Enables plugin for transform the RegExp literal has `v` flag
+    pub set_notation: bool,
+}
+
+impl RegExpOptions {
+    pub fn with_sticky_flag(&mut self, value: bool) -> &Self {
```

Comment:
Do we need these builder builds? given all the fields are public already.

Reply:
I haven't used them, i will remove all.